mpifileutils.spec

```diff
@@ -73,6 +73,11 @@ BuildRequires: openssl-devel
 BuildRequires: gcc-c++
 BuildRequires: libuuid-devel
 
+
+%if (0%{?suse_version} >= 1500)
+BuildRequires: libfabric1 >= 1.12.0
+%endif
+
```

Comment:
This was a hack that I applied to get the rpm to build on leap15. There seemed to be some sort of dependency issue in another package - ideally I don't think we should need this, so I'm not sure how to handle it